packages/extension-ui/src/util/clearClipboard.ts

```diff
@@ -12,3 +11,8 @@ export default () => {
   form.blur();
   document.body.removeChild(form);
 };
+
+export default () => {
+  navigator.clipboard.writeText(' ').catch(() => null);
+  clearClipboardFallback();
```

Comment:
Maybe rather:
```suggestion
  navigator.clipboard.writeText(' ').catch(() => clearClipboardFallback());
```
?

Reply:
It makes a lot of sense, I was just freaking out that if I use async and wait for result my narrow window of opportunity to interact with clipboard after user interaction will pass and everything will be lost :scream:

I'll change it :+1: